modules/storage/src/main/kotlin/edu/stanford/spezi/modules/storage/secure/Credentials.kt

```diff
@@ -0,0 +1,6 @@
+package edu.stanford.spezi.modules.storage.secure
+
+data class Credentials(
```

Comment:
Is there something against adding server as a nullable parameter of this type?

Reply:
no, I would then also add accessGroup to it on iOS, hope that makes sense.